src/app/components/shared/event-list/event-list.ts

```diff
@@ -153,6 +160,18 @@ export class EventList implements OnInit {
     });
   }
 
+  bulkPositionUpdate(column, direction) {
+    let params = { status: this.status.key, column: column, direction: direction }
+    this.eventCollectionsAPI
+        .updatePosition(this.projectId, params)
+        .subscribe(
+          (resp: any) => {
+            this.events = resp.event_collections;
+            this.eventCount = resp.meta.total_count;
+          },
+          this.onGetEventsError);
```

Comment:
alternate indentation
```suggestion
        .subscribe((resp: any) => {
          this.events = resp.event_collections;
          this.eventCount = resp.meta.total_count;
        }, this.onGetEventsError);
```